subprojects/docs/src/docs/userguide/ant.adoc

```diff
@@ -27,173 +27,271 @@ You can use Gradle's Ant integration as a path for migrating your build from Ant
 
```

Comment:
An observation: the footnote above at line 24 is now labeled as "1" while previously it showed as "5".  Seems like the footnote number is probably being tracked per-page now instead of across the whole userguide.

Reply:
Yes, and I think this is an improvement :) Agreed?

Reply:
All good.